client/extensions/woocommerce/app/settings/shipping/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import ShippingLabels from './shipping-labels';
+import ShippingOrigin from './shipping-origin';
+import ShippingPackageList from './shipping-package-list';
+import ShippingZoneList from './shipping-zone-list';
+
+const Shipping = ( { className } ) => {
```

Comment:
Please add propTypes for className (and soon enough I expect we will have more), e.g.: https://github.com/Automattic/wp-calypso/blob/master/client/extensions/woocommerce/store-sidebar/store-ground-control.js#L40